web/src/styles/wiki.css

```
@@ -1152,15 +1180,15 @@
   opacity: 0.7;
 }
 .wk-section-new {
-  font-family: var(--wk-mono);
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)